optuna_integration/_lightgbm_tuner/_train.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+from collections.abc import Callable
+from typing import Any
+
+from optuna._imports import try_import
+from optuna.integration._lightgbm_tuner.optimize import _imports
+from optuna.integration._lightgbm_tuner.optimize import LightGBMTuner
+from optuna.study import Study
+from optuna.trial import FrozenTrial
+
+
+with try_import():
+    import lightgbm as lgb
+
+
+def train(
+    params: dict[str, Any],
+    train_set: "lgb.Dataset",
+    num_boost_round: int = 1000,
+    valid_sets: list["lgb.Dataset"] | tuple["lgb.Dataset", ...] | "lgb.Dataset" | None = None,
+    valid_names: Any | None = None,
+    feval: Callable[..., Any] | None = None,
+    feature_name: str = "auto",
+    categorical_feature: str = "auto",
+    keep_training_booster: bool = False,
+    callbacks: list[Callable[..., Any]] | None = None,
+    time_budget: int | None = None,
+    sample_size: int | None = None,
+    study: Study | None = None,
+    optuna_callbacks: list[Callable[[Study, FrozenTrial], None]] | None = None,
+    model_dir: str | None = None,
+    verbosity: int | None = None,
+    show_progress_bar: bool = True,
+    *,
+    optuna_seed: int | None = None,
+) -> "lgb.Booster":
+    """Wrapper of LightGBM Training API to tune hyperparameters.
+
+    It optimizes the following hyperparameters in a stepwise manner:
+    ``lambda_l1``, ``lambda_l2``, ``num_leaves``, ``feature_fraction``, ``bagging_fraction``,
+    ``bagging_freq`` and ``min_child_samples``.
+    It is a drop-in replacement for `lightgbm.train()`_. See
+    `a simple example of LightGBM Tuner <https://github.com/optuna/optuna-examples/tree/main/
+    lightgbm/lightgbm_tuner_simple.py>`_ which optimizes the validation log loss of cancer
+    detection.
+
+    :func:`~optuna.integration.lightgbm.train` is a wrapper function of
+    :class:`~optuna.integration.lightgbm.LightGBMTuner`. To use feature in Optuna such as
+    suspended/resumed optimization and/or parallelization, refer to
+    :class:`~optuna.integration.lightgbm.LightGBMTuner` instead of this function.
```

Comment:
```suggestion
    :func:`~optuna_integration.lightgbm.train` is a wrapper function of
    :class:`~optuna_integration.lightgbm.LightGBMTuner`. To use feature in Optuna such as
    suspended/resumed optimization and/or parallelization, refer to
    :class:`~optuna_integration.lightgbm.LightGBMTuner` instead of this function.
```